src/components/core/ImagePicker.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import {Button, Image, View, Platform, Pressable} from 'react-native';
+import styled from 'styled-components/native'
+import * as ExpoImagePicker from 'expo-image-picker';
+
+// @ts-ignore
+//import noImage from '../../assets/images/noImage.png'
+
+interface ImagePickerProps {
+    setImage: Function
+    image: string
+}
+
+const ImagePicker = ({
+        setImage,
+        image
+    }: ImagePickerProps) => {
+    useEffect(() => {
+        (async () => {
+            if (Platform.OS !== 'web') {
+                const { status } = await ExpoImagePicker.requestMediaLibraryPermissionsAsync();
+                if (status !== 'granted') {
+                    alert('Sorry, we need camera roll permissions to make this work!');
+                }
+            }
+        })();
+    }, []);
+
+    const pickImage = async () => {
+        let result: any = await ExpoImagePicker.launchImageLibraryAsync({
+            mediaTypes: ExpoImagePicker.MediaTypeOptions.All,
```

Comment:
You probably need only images here. And also it would make sense to not allow editing here.